sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client_models.hpp

```diff
@@ -187,6 +239,102 @@ namespace Azure { namespace Security { namespace Attestation { namespace Models
     std::vector<AttestationSigner> Signers;
   };
 
+  /** An AttestationToken represents an RFC 7519 JSON Web Token returned from the attestation
+   * service with the specialized body type.
+   * <typeparam name="T"></typeparam> The type which represents the body of the attestation token.
+   */
+  template <typename T> class AttestationToken final {
+  public:
+    /**
+     * @brief The full RFC 7515 JWS/JWT token returned by the attestation service.
+     */
+    std::string RawToken;
+
+    /**
+     * @brief The elements of the raw token which will be signed by the Signature.
+     */
+    std::string SignedElements;
+
+    /**
+     * @brief Signature (if present) for the attestation token.
+     */
+    std::vector<uint8_t> Signature;
+
+    /**
+     * @brief RFC 7515 header properties.
+     */
+    Models::AttestationTokenHeader Header;
+
+    // RFC 7519 properties.
+
+    /**
+     *  The Expiration time for this attestation token.
+     *
+     * After this time, the token cannot be considered valid.
+     *
+     * See <a href='https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4'>RFC 7519
+     * Section 4.1.4</a> for more information.
```

Comment:
Just curious, does it render as we expect in the generated doxygen docs?

Reply:
What about this wouldn't render properly?